vignettes/howto-dm-rows.Rmd

```diff
@@ -16,8 +16,6 @@ editor_options:
 source("setup/setup.R")
 ``````
 
-## Introduction {#intro}
```

Comment:
Removing this section header to be consistent with most of the other vignettes that don't have it.